test/src/unit-sparse-unordered-with-dups-reader.cc

```diff
@@ -1821,6 +1821,29 @@ TEST_CASE_METHOD(
   tiledb_array_t* array = nullptr;
   tiledb_query_t* query = nullptr;
 
+  // Disable merge overlapping sparse ranges.
+  // Support for returning multiplicities for overlapping ranges will be
+  // deprecated in a few releases. Turning off this setting allows to still
+  // test that the feature functions properly until we do so. Once support is
+  // fully removed for overlapping ranges, this section can be deleted.
```

Comment:
Change the last sentence to:
Once support is fully removed for overlapping ranges, num_dups can be removed and read_strings modified to remove multiplicities.